Source/WebCore/rendering/RenderBox.h

```diff
@@ -822,6 +826,11 @@ inline RenderBox* RenderBox::firstChildBox() const
     return nullptr;
 }
 
+inline RenderBox* RenderBox::firstInFlowChildBox() const
+{
+    return downcast<RenderBox>(firstInFlowChild());
```

Comment:
what guarantees that the first inflow child is always a RenderBox? (or you meant to call dynamicDowncast?)

Reply:
Yes dynamicDowncast is what should have been used here. Changed.